visallo/visallo
# Fix errors identified by the graph check cli

Two main sources of errors that were identified by the graph check cli:
1. Properties were not being set with the proper metadata for many ontology vertices. Fixing this required providing a user to more of the ontology methods so that they could build the proper metadata entries.
2. Ontology code generation was not setting the visibility json property on entities and relationships

Testing Instructions:
1. Start a new Visallo instance
2. Ingest data with the code generation api
3. Run the graph check cli tool and verify that there are no warnings/errors reported

CHANGELOG
Changed: The IngestRepository will now set VisalloProperties.VISIBILITY_JSON on the entities and relationships that it creates.
